### PR TITLE
Add messages for unpublish

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: Model/Content/MessageHandler/UnpublishContentMessageHandler.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        ContentDimensionRepositoryInterface $contentDimensionRepository,
+        DimensionIdentifierRepositoryInterface $dimensionIdentifierRepository
+    ) {
+        $this->contentDimensionRepository = $contentDimensionRepository;
+        $this->dimensionIdentifierRepository = $dimensionIdentifierRepository;
+    }
+
+    public function __invoke(UnpublishContentMessage $message): void
+    {
+        $this->unpublishContentDimension($message->getResourceKey(), $message->getResourceId());
```
Comment (nnatter) [inline Model/Content/MessageHandler/UnpublishContentMessageHandler.php]: this way we will unpublish "parts" of other locales too?

File: Model/Excerpt/MessageHandler/UnpublishExcerptMessageHandler.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        ExcerptDimensionRepositoryInterface $excerptDimensionRepository,
+        DimensionIdentifierRepositoryInterface $dimensionIdentifierRepository
+    ) {
+        $this->excerptDimensionRepository = $excerptDimensionRepository;
+        $this->dimensionIdentifierRepository = $dimensionIdentifierRepository;
+    }
+
+    public function __invoke(UnpublishExcerptMessage $message): void
+    {
+        $this->unpublishExcerptDimensions($message->getResourceKey(), $message->getResourceId());
```
Comment (nnatter) [inline Model/Excerpt/MessageHandler/UnpublishExcerptMessageHandler.php]: same as above

File: Model/Seo/MessageHandler/UnpublishSeoMessageHandler.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        SeoDimensionRepositoryInterface $seoDimensionRepository,
+        DimensionIdentifierRepositoryInterface $dimensionIdentifierRepository
+    ) {
+        $this->seoDimensionRepository = $seoDimensionRepository;
+        $this->dimensionIdentifierRepository = $dimensionIdentifierRepository;
+    }
+
+    public function __invoke(UnpublishSeoMessage $message): void
+    {
+        $this->unpublishSeoDimensions($message->getResourceKey(), $message->getResourceId(), $message->getLocale());
```
Comment (nnatter) [inline Model/Seo/MessageHandler/UnpublishSeoMessageHandler.php]: but you did not do it here? 😂

Comment (wachterjohannes) [reply Model/Seo/MessageHandler/UnpublishSeoMessageHandler.php]: the seo does not have any untranslated properties

**Pull request merged**